mlflow/utils/thread_utils.py

```diff
@@ -0,0 +1,36 @@
+import os
+import threading
+
+_thread_locals = threading.local()
+_globals = {}
+_globals_lock = threading.Lock()
```

Comment:
What are these? Where are these used?

Reply:
oh sorry. These are not used. I will clean them .